### PR TITLE
Feature home page

### PR DESCRIPTION
**Pull request by okikiola11**
#### What does this PR do?
- Shows how to create HTML Forms

#### Description of Task to be completed?
- Added the header tag
- Added the main section tag which contains the various input boxes and elements.
- Added the footer.
- Ran validations on our Html and CSS.

#### How should this be manually tested?
- Clone this branch.
- open the index.html file on your browser

#### Screenshot
![screenshot]
screenshot

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
### Kudos for great descriptive Pull Request :1st_place_medal: 
* [ ] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows one error in your project.
* [ ] stickler is present but not setup correctly, so not checking your project yet. Take a look [here](https://github.com/microverseinc/linters-config/tree/master/css) to configure your stickler. Right now it doesn't have the right content.
* [ ] Images are not displaying correctly right now. Row under intuit logo should contain Mint logo and text, Quickbooks logo an text,  Turbotax logo and text  (logo on the left, text on the right) ([link to page](https://accounts.intuit.com/signup.html))
image

* [ ] Phone `<input>` should be of type `tel` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))
* [ ] Your form is missing a disclaimer text that matches the context and style of the original page ([link to page](https://accounts.intuit.com/signup.html))
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <div class="forgot-pwd-text">Already have an intuit account?</div>
+            <a href="">I forgot my user ID or password</a>
+        </div>
+        <button class="signin-button bg-blue"> <i class="fa fa-lock"></i> Sign In</button>
+    </header>
+
+    <main>
+        <section class="container">
+            <div class="flexbox logo-img">
+                <img src="assets/images/logo.png" alt="Company Logo"/>
```
Comment (euqueme) [inline index.html]: change `src="assets/images/logo.png"` to: 

- `src="assets/images/logo.PNG"`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            </div>
+
+            <div class="flexbox icon-container"> 
+                <div class="icon-container-logo">
+                    <img src="assets/images/icon-1.png" alt="Mint Icon"/>
```
Comment (euqueme) [inline index.html]: change `src="assets/images/icon-1.png"` to:

- `src="assets/images/icon-1.PNG"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                </div>   
+                <div class="icon-container-txt">mint</div>
+
+                <div class="icon-container-logo">
+                    <img src="assets/images/icon-2.png" alt="Quickbook Icon"/>
```
Comment (euqueme) [inline index.html]: change `src="assets/images/icon-2.png"` to:

- `src="assets/images/icon-2.PNG"`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                </div>  
+                <div class="icon-container-txt">quickbook</div>
+
+                <div class="icon-container-logo">
+                    <img src="assets/images/icon-3.png" alt="Turbotax Icon"/>
```
Comment (euqueme) [inline index.html]: change `src="assets/images/icon-3.png"` to:

- src="assets/images/icon-3.PNG"

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @okikiola11 and @frankopkusianwar :balloon:  :lollipop: 

### Thank you for doing the changes required by the previous reviewer Good job! :medal_sports:  your project looks almost like the original :microscope: 

### you just have to make a couple more changes and some optional ones if you'd like :point_down: 

## Follow up

**Broken images**

- [ ] The images are still broken on your page, change the extensions from `.png` to `.PNG` I have commented your code where changes have to be made :arrow_double_down: 

- this is how it looks in the inspector with the change:

image

**Page structure**

- [ ] The order of the words in the original page is: **turbotax**, **quickbox** and **mint** and yours is **mint**, **quickbooks** and **turbotax**, the logos are in the right position so let's rearrange the words:

- Yours

image

- Original

image

## Optional

- [ ] [OPTIONAL]your Live Demo leads to a not found page, fix it or remove it from your Readme file
- [ ] [OPTIONAL] I encourage the use of  the `<nav>` tag for the major navigation links

### That's it team! :raised_hands: 

### Please :pray:  do the changes :memo: and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any questions? ask me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Good job so far and making the review changes :clap: and your project looks good. Congratulations :boom: :100: I approve of this project.
### You can merge and submit a project completion form from your dashboard. Good luck with the next project.
- Happy coding!

**Pull request merged**